### DISCUSSION TITLE
.NET Conf 2024 demo of Hot Design?

### DISCUSSION THREAD

**Discussion by ward-horsfall** [unoplatform/uno#18817] [Q&A] [answered]
Hi,

Is there a URL of this video as I have searched through them all and cant find it?

https://platform.uno/blog/uno-platform-studio-featuring-hot-design/

It seems to not be available on YouTube.

Here is the announcement on the .NET conf agenda:

Announcing Uno Platform Studio and Hot Design
15 November - 9:00 am - 9:30 am AEDT

https://www.dotnetconf.net/agenda

Thanks

Ward

**Comment by jeromelaban** [comment, marked as answer]
We're still waiting for the video to be updated on youtube. We've been told it will be early this week.

**Reply by morning4coffe-dev** [reply]
@ward-horsfall @jeromelaban The recording is now available! https://youtu.be/sJPyieyt1Rc
